pyperformance/data-files/benchmarks/bm_docutils/run_benchmark.py

```diff
@@ -0,0 +1,57 @@
+"""
+Convert Docutils' documentation from reStructuredText to <format>.
+"""
+
+import contextlib
+from pathlib import Path
+import time
+
+import docutils
+from docutils import core
+import pyperf
+
+try:
+    from docutils.utils.math.math2html import Trace
+except ImportError:
+    pass
+else:
+    Trace.show = lambda message, channel: ...  # don't print to console
+
+DOC_ROOT = Path("data/docs").resolve()
```

Comment:
```suggestion
DOC_ROOT = (Path(__file__).parent / "data" / "docs").resolve()
```

This seems to fix the failure for me.  Without this, if `pyperformance` is run from any directory other than the one containing this file, the relative path is wrong, therefore it doesn't find any files, and the benchmark ends up taking 0 seconds.

Reply:
Of course! Thanks for looking into this, I really appreciate it.

A